demo/bulk_insert/bulk_insert.py

```diff
@@ -118,9 +135,18 @@ def ProcessNodes(nodes_csv_files):
 			# The first row of a label CSV contains its attributes
 			descriptor.attributes = header_row
 
+
 			# Update counts
 			header_len = len(header_row)
 			descriptor.attribute_count += header_len
```

Comment:
It would be much cleaner if the Descriptor constructor would set its attributes and attribute_count by accepting the header_row as a parameter.